packages/api/config/api_v1.yml

```diff
@@ -477,3 +477,20 @@ ApiBulkDeleteV1:
       value: AsyncOperationTaskDefinition
     stackName: '{{parent.stackName}}'
     systemBucket: '{{parent.bucket}}'
+
+ApiInstanceMetaV1:
+  handler: index.instanceMeta
+  apiRole: true
+  source: 'node_modules/@cumulus/api/dist/'
+  envs:
+    cmr_provider: '{{parent.cmr.provider}}'
```

Comment:
Not a requested change, but what does this actually do?

Reply:
This the PR? or this the env.cmr_provider?  
This line here injects the cmr provider from the example/app/config.yml https://github.com/nasa/cumulus/blob/master/example/app/config.yml#L126 into the environment so that it's available for the lambda at process.env.cmr_provider.

Reply:
Yeah, I meant what this line was doing. Thanks for explaining